bigbluebutton-html5/package.json

```
@@ -175,4 +175,4 @@
       "pre-commit": "lint-staged"
     }
   }
-}
+}
```

```suggestion
}

```
Just the trailing new line on this  file